cytotable/convert.py

```diff
@@ -399,17 +417,42 @@ def _source_pageset_to_parquet(
 
     from cytotable.utils import (
         _duckdb_reader,
+        _extract_npz_to_parquet,
         _sqlite_mixed_type_query_to_parquet,
         _write_parquet_table_with_metadata,
     )
 
+    source_type = str(source["source_path"].suffix).lower()
+
     # attempt to build dest_path
     source_dest_path = (
         f"{dest_path}/{str(AnyPath(source_group_name).stem).lower()}/"
         f"{str(source['source_path'].parent.name).lower()}"
     )
     pathlib.Path(source_dest_path).mkdir(parents=True, exist_ok=True)
 
+    # If we have npz files, we need to extract them in a specialized manner.
+    # See below for CSV and SQLite handling.
+    if source_type == ".npz":
+        return _extract_npz_to_parquet(
+            source_path=str(source["source_path"]),
+            dest_path=f"{source_dest_path}/{str(source['source_path'].stem)}.parquet",
+            tablenumber=source["tablenumber"],
+        )
+
+    if pageset is None:
```

Comment:
Kind of a silly thing, but maybe make this an `elif` chained to the above `if`, so it's obvious that this is related to it not being an NPZ? It's functionally the same, but might be more readable if someone's skimming and didn't see the return above.

Reply:
Makes sense to me, thanks! I agree this is clearer - I've gotten into a habit lately of using `elif` less and less but this is a spot where it would indeed increase readability.